apis/projectcontour/v1alpha1/extensionservice.go

```diff
@@ -20,11 +20,11 @@ import (
 
 // ExtensionProtocolVersion is the version of the GRPC protocol used
 // to access extension services. The only version currently supported
-// is "v2".
+// is "v3".
 type ExtensionProtocolVersion string
 
-// SupportProtocolVersion2 requests the "v2" support protocol version.
-const SupportProtocolVersion2 ExtensionProtocolVersion = "v2"
+// SupportProtocolVersion3 requests the "v3" support protocol version.
+const SupportProtocolVersion3 ExtensionProtocolVersion = "v3"
```

Comment:
I assumed that since this is in the `v1alpha` API, we can make a breaking change here by renaming it/changing the default value.

Reply:
My preference would be to no make a breaking change unless it's really required. Now we are in the v3 API, we can simply send the version in the proto message.

Reply:
I changed this to retain the v2 constant, but as of now this code is still statically programming Envoy to v3.